uwblueprint/ccbc
# [Feat] Add search results number

## Notion ticket link

[Search Results Number](https://www.notion.so/uwblueprintexecs/Search-Results-Number-852ef03f02804cc0ba228dc63aa4cf34)



## Implementation description
*  Added state for the search results number and display it on the search reviews page



## Steps to test
1. verify that there is no search results number on the page without searching something up first
2. search up a keyword (i.e. Harry, Hatch, Kingdom)
3. verify that the search results number is right



## What should reviewers focus on?
*  there is no search results number without search


## Checklist
